db/db_test.cc

```diff
@@ -3512,8 +3512,14 @@ TEST_F(DBTest, SanitizeNumThreads) {
                      (i < 4) ? Env::Priority::LOW : Env::Priority::HIGH);
     }
 
-    // Wait 100 milliseconds for they are scheduled.
-    env_->SleepForMicroseconds(100000);
+    // Wait until 10s for they are scheduled.
```

Comment:
I thought about something like that. But complicates the code but is still a poll-based solution so it doesn't make a fundamental difference. Furthermore, the failure of the test will show up with timeout, rather than actual assert failure, which is arguable whether it is better.